docker/testing.entrypoint.sh

```diff
@@ -63,13 +62,11 @@ if [ ! -f "$PROJECT_DIR/c3.php" ]; then
     curl -L 'https://raw.github.com/Codeception/c3/2.0/c3.php' > "$PROJECT_DIR/c3.php"
 fi
 
-if [[ -n "$LOWEST" ]]; then
-    PREFER_LOWEST="--prefer-source"
-fi
-
 # Install dependencies
-COMPOSER_MEMORY_LIMIT=-1 composer update --prefer-source ${PREFER_LOWEST}
```

Comment:
There is plenty of google/stackoverflow about composer with --prefer-source vs --prefer-dist.

Reply:
If we ever think we will use the --prefer-lowest of packages versions, let me know ;) otherwise I removed the LOWEST flag.